### PR TITLE
Fixes #1582 - ClayCSS 2.x SVG Icons added new icons `color-picker`, `spac…

### LINKED ISSUE #1582: ClayCSS add new SVG Icons Color Picker, Spacer, User Plus
https://issues.liferay.com/browse/LEXI-516
https://issues.liferay.com/browse/LEXI-518
https://issues.liferay.com/browse/LEXI-519

### PR DESCRIPTION
**Pull request by pat270**
…er`, and `user-plus`

### CODE REVIEW

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: packages/clay-css/src/images/icons/spacer.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg id="spacer" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
```
Comment (matuzalemsteles) [inline packages/clay-css/src/images/icons/spacer.svg]: hey @pat270 Spacer has some broken property... can you check it?

screen shot 2019-02-21 at 13 58 16

Comment (matuzalemsteles) [reply packages/clay-css/src/images/icons/spacer.svg]: I think we only have the class declaration twice in the `rect` tag.
```
class="lexicon-icon-outline spacer-indicator-start"
class="st0"
```

**Comment by pat270** [comment]
@matuzalemsteles I cleaned up the extra class attribute. Thanks!

**Pull request merged**